package.json

```diff
@@ -34,10 +35,32 @@
   "readmeFilename": "README.md",
   "license": "MIT",
   "devDependencies": {
-    "browserify": "^13.3.0",
+    "browserify": "^14.3.0",
     "localStorage": "^1.0.3",
     "mocha": "3.0.2",
     "socketcluster-server": "^6.3.0",
-    "uglify-js": "^2.8.29"
+    "uglify-js": "^2.7.5",
+    "babel-jest": "^19.0.0",
+    "babel-plugin-flow-comments": "^6.3.19",
+    "babel-plugin-inline-package-json": "~2.0.0",
+    "babel-plugin-minify-dead-code-elimination": "0.1.4",
+    "babel-plugin-transform-inline-environment-variables": "^6.8.0",
+    "babel-plugin-transform-runtime": "^6.15.0",
+    "babel-preset-es2015": "^6.14.0",
+    "babel-preset-react": "^6.11.1",
+    "babel-preset-stage-2": "^6.13.0",
+    "codecov.io": "^0.1.6",
```

Comment:
@odenktools That's a LOT of dev dependencies. Is it possible to remove some of them? Personally, I'm not a big fan of code coverage tools; I think it encourages people to write lots of tedious tests which cover basic functionality. 100% coverage means nothing if you don't test the right parts of the code with the right inputs. Too many tests make it harder to update the code because you have to maintain all the tests.

Reply:
@jondubois thanks for reviews... sorry my bad code.. yes its possible, wait ok

Reply:
@jondubois ok done! you can review again.. and also the output size more smaller than using browserify